rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -24,24 +25,15 @@ if [ -f /etc/ld.so.conf.d/rudder.conf ]; then
   ldconfig -v >> /var/log/rudder/install/rudder-agent.log 2>&1
 fi
 
-echo "Making sure that the permissions on the CFEngine key directory are correct..."
-if [ -d /var/rudder/cfengine-community/ppkeys ]; then
-  chmod 700 /var/rudder/cfengine-community/ppkeys
-  if [ `ls /var/rudder/cfengine-community/ppkeys | wc -l` -gt 0 ]; then
-    chmod 600 /var/rudder/cfengine-community/ppkeys/*
-  fi
-fi
-
-# Copy new binaries to workdir, make sure daemons are stopped first
-
 # Set a "lock" to avoid CFEngine being restarted during the upgrade process
 I_SET_THE_LOCK=0
 if [ ! -f /opt/rudder/etc/disable-agent ]; then
   I_SET_THE_LOCK=1
-  touch /opt/rudder/etc/disable-agent
+  ${RUDDER_CMD}/agent-disable
```

Comment:
Why are you calling the commands directly in /opt/rudder/share/commands rather than via their wrapper (/opt/rudder/bin/rudder)?

I think this could backfire on us one day, since that wrapper could very well add some sanity checks, or other features, and the individual commands may not function standalone.